web/src/Proposal.jsx

```diff
@@ -1,26 +1,15 @@
 import React from "react";
 import {
-  TableComposable,
-  Thead,
-  Tr,
-  Th,
-  Tbody,
-  Td
-} from "@patternfly/react-table";
-
-import filesize from "filesize";
```

Comment:
Remember to remove the _filesize_ dependency, no longer needed.

Reply:
Good point.